sonar-db/src/main/resources/org/sonar/db/permission/template/PermissionTemplatePermissionMapper.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="org.sonar.db.permission.template.PermissionTemplatePermissionMapper">
+  <sql id="columns">
+    ptp.id,
+    ptp.template_id as templateId,
+    ptp.permission_key as permission,
+    ptp.with_project_creator as withProjectCreator,
+    ptp.created_at as createdAt,
+    ptp.updated_at as updatedAt
+  </sql>
+
+  <select id="selectByTemplateId" parameterType="long" resultType="TemplatePermission">
+    select
+    <include refid="columns" />
+    from perm_tpl_permissions ptp
+    where ptp.template_id=#{templateId}
```

Comment:
When validating the schema with Atlas, tell them that you need to query this table by template_id only (as there's no index on it)